prime-react-demo/src/main/scala/react/table/demo/TreeDemo.scala

```diff
@@ -49,6 +51,15 @@ object TreeDemo:
             )
           )
         ),
-        Tree(nodes.value, nodeTemplate = (i, _) => <.span(<.b("Hello there, "), i).rawNode)
+        Tree(
+          nodes.value,
+          selectionMode = Tree.SelectionMode.Single,
+          nodeTemplate = (i, _) => <.span(<.b("Hello there, "), i).rawNode,
+          onSelect = (d, e) =>
+            Callback {
+              println(s"selected $d")
+              js.Dynamic.global.console.log(e)
```

Comment:
For future reference: there's a `Callback.log` method that does this (and `.info` and `.warn`).